CODE_REVIEW.md

```diff
@@ -0,0 +1,73 @@
+# Code review requiements
+
+This document describe responsibilities of code reviewers when reviewing PRs.
+
+Pull request process is described in [contributing guide](./CONTRIBUTING.md#submitting-pull-requests)
+
+## Base checklist
+
+- All automated test steps pass (e.g. tests, lints, build)
+- PR title follows [commit conventions](CONTRIBUTING.md#commit-conventions)
+    - If PR format is different, reviewer should change it to follow the conventions
+- PR has a description with reasoning and change overview
+    - If description is missing but clear for reviewer, reviewer may add a comment or edit description
+- New feature or fix has tests proving it works
+    - Reviewer should request changes from contributor to add tests
+- If PR changes documented behaviour, there should be update to `/docs` or inline comments
```

Comment:
It took me a second to realise what exactly we meant by `If PR changes documented behaviour`. Should we try to simplify it.
`If the PR changes a behaviour that is documented in our public docs`?
This should be applicable for something that is being  added (not updated)  and the documentation needs to be added for that, I think we should specify this as well.